docs/src/content/docs/es/challenges/angular/4-context-outlet-typed.md

```diff
@@ -0,0 +1,46 @@
+---
+title: 🔴 ContextOutlet en forma de tipo
+description: El desafío 4 se trata de tipificar de manera fuerte las directivas de ngContextOutlet
+author: thomas-laforge
+challengeNumber: 4
+command: angular-context-outlet-type
+blogLink: https://medium.com/@thomas.laforge/ngtemplateoutlet-type-checking-5d2dcb07a2c6
+sidebar:
+  order: 201
+---
+
+## Información
+
+:::note[Nota]
+
+Las declaraciones de tipos permanecerán en ingles, asi como los nombres de los objetos y sus miembros.
+
+:::
+
+Angular ofrece la function estática [`ngTemplateContextGuard`](https://angular.io/guide/structural-directives#typing-the-directives-context) para reforzar el tipo de una directiva estructural.
+
+Sin embargo, el contexto de **NgTemplateOutlet** es de tipo **Object**. Con la ayuda de la guardia mencionada anteriormente, podemos mejorar ese comportamiento.
+
+## Declaración
+
+En este desafío, queremos aprender como reforzar el tipo de nuestro ng-template en AppComponent.
+
+Este desafío ofrece dos niveles de complejidad:
+
+### Nivel 1: una Interfase conocida
+
+A continuación tenemos estas lineas de código:
+
+!['Unkown' en 'Person'](../../../../assets/4/unknown-person.png 'Unkown' en 'Person')
```

Comment:
Can we take a look at this? looks like the path needs updating since the `img` is not being shown...Its probably an extra `../` becuase of the es folder it lives on

<img width="1229" alt="image" src="https://github.com/tomalaforge/angular-challenges/assets/62297014/22a83d17-5890-4eb0-8256-c6fbbc1ed491">


english:
<img width="1064" alt="image" src="https://github.com/tomalaforge/angular-challenges/assets/62297014/a42e412a-1168-4c18-86fb-3ea8a70c8ea7">

Reply:
huh... when I run in Dev, even with the correct path, it is giving me the same kind of issue. I was able to fix it like this:

```md
![Any en Person](../../../../../assets/4/unknown-person.png)
```

but as a result, we won't have the tag title on the image. for some reason we can't push images with titles on it.

Reply:
now it looks like this:
![image](https://github.com/tomalaforge/angular-challenges/assets/1978642/e9f191c1-cd07-499b-9005-aa871065c765)
Probably we need to add a way to support a caption on the image?